client/src/components/Search.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SortOptions = styled.div`
+  display: flex;
+  text-align:right;
+  margin:0px auto 0px auto;
+`;
+
+const SearchContainer = styled.div`
+  display: flex;
+  max-width: 700px;
+`;
+
+class Search extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      sortOption: 0
+    };
+    this.executeSearch = this.executeSearch.bind(this);
+    this.onSearchChange = this.onSearchChange.bind(this);
+    this.onSortChange = this.onSortChange.bind(this);
+  }
+
+  onSearchChange(e) {
+    this.setState({
+      searchQuery: e.target.value,
+    });
+  }
+
+  executeSearch() {
+    this.props.search(this.state);
```

Comment:
Did you mean to execute the search on the entire state of the component?

Reply:
Yes. If I build out more search parameters and include them in the state, this function can remain the same.